src/backend/src/utils/change-requests.utils.ts

```diff
@@ -27,33 +27,71 @@ export const sendSlackChangeRequestNotification = async (
   message: string,
   crId: number,
   budgetImpact?: number
-) => {
-  if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
-  const msgs = [];
+): Promise<{ channelId: string; ts: string }[]> => {
+  if (process.env.NODE_ENV !== 'production') return []; // don't send msgs unless in prod
+  const msgs: { channelId: string; ts: string }[] = [];
   const fullMsg = `:tada: New Change Request! :tada: ${message}`;
   const fullLink = `https://finishlinebyner.com/cr/${crId}`;
   const btnText = `View CR #${crId}`;
-  msgs.push(sendMessage(team.slackId, fullMsg, fullLink, btnText));
+  const notification = await sendMessage(team.slackId, fullMsg, fullLink, btnText);
+  if (notification) msgs.push(notification);
 
   if (budgetImpact && budgetImpact > 100) {
-    msgs.push(
-      sendMessage(process.env.SLACK_EBOARD_CHANNEL!, `${fullMsg} with $${budgetImpact} requested`, fullLink, btnText)
+    const importantNotification = await sendMessage(
+      process.env.SLACK_EBOARD_CHANNEL!,
+      `${fullMsg} with $${budgetImpact} requested`,
+      fullLink,
+      btnText
     );
+    if (importantNotification) msgs.push(importantNotification);
   }
-  return Promise.all(msgs);
+
+  return msgs;
 };
 
 export const sendSlackCRReviewedNotification = async (slackId: string, crId: number) => {
   if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
   const msgs = [];
-  const fullMsg = `:tada: Your Change Request was just reviewed! Clink the link to view! :tada:`;
+  const fullMsg = `:tada: Your Change Request was just reviewed! Click the link to view! :tada:`;
   const fullLink = `https://finishlinebyner.com/cr/${crId}`;
   const btnText = `View CR#${crId}`;
   msgs.push(sendMessage(slackId, fullMsg, fullLink, btnText));
 
   return Promise.all(msgs);
 };
 
+export const sendSlackCRStatusToThread = async (slackId: string, crId: number, ts: string, approved: boolean) => {
+  if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
+  const msgs = [];
+  const fullMsg = `This Change Request was ${approved ? 'approved! :tada:' : 'denied.'} Click the link to view.`;
+  const fullLink = `https://finishlinebyner.com/cr/${crId}`;
+  const btnText = `View CR#${crId}`;
+  msgs.push(replyToMessageInThread(slackId, ts, fullMsg, fullLink, btnText));
```

Comment:
so I think we can actually await the response and just return it like you did elsewhere. I think the msgs/ promise.all stuff was just copied over here when earlier code was written